composer.json

```diff
@@ -22,6 +22,7 @@
         "psr/http-message-implementation": "~1.0",
         "ramsey/uuid": "~3.3",
         "symfony/options-resolver": "~2.7|~3.0",
+        "symfony/property-access": "~2.7|3.0",
```

Comment:
I'm not a fan of adding this additional dependency, but it's due to the fact that I don't think that the functionality is helpful: reaching data by path seems helpful, but it's still an array underneath, and I don't see any reason for changing that; so, accessing by array keys would be easier and more natural (and maybe it allows some auto completing from the IDE).

Reply:
The problem is that if you have a complex array with nested arrays inside, even autocompleting from IDE does not help you and the only way to reach a specific subkey would be to get the whole array and then access the key you want. If you also have to change the value, you would have to put inside the context object the whole array. But I'm not a big fan of adding an additional dependency either, so if we all agree this is something not useful I'm all in favour of removing it